omnisafe/adapter/online_adapter.py

```diff
@@ -169,38 +172,42 @@ def observation_space(self) -> OmnisafeSpace:
     def step(
         self,
         action: torch.Tensor,
-    ) -> tuple[torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor, dict]:
+    ) -> tuple[
+        torch.Tensor,
+        torch.Tensor,
+        torch.Tensor,
+        torch.Tensor,
+        torch.Tensor,
+        dict[str, Any],
+    ]:
         """Run one timestep of the environment's dynamics using the agent actions.
 
         Args:
-            action (torch.Tensor): action.
+            action (torch.Tensor): action from the agent or random.
```

Comment:
```suggestion
            action (torch.Tensor): The action from the agent or random.
```

Should be a sentence and capitalize the first letter.